### PR TITLE
Update ghost chart to work with non-root ghost container

### PR DESCRIPTION
**Pull request by tompizmor**
Since ghost 0.11.10-r2 version, it is being used a non-root approach to run the container so we need to set permission in the volume before initializing the ghost application.

### CODE REVIEW

File: stable/ghost/templates/deployment.yaml
Comment (dhilipkumars) [inline stable/ghost/templates/deployment.yaml]: this needs to be from some lime below in `value.yaml`
```
volumePermision:
    image: 
         name: <IMAGE NAME>
         tag:  <IMAGE TAG>
         pullPolicy: <IMAGE PULL POLICY>
```

**Review by dhilipkumars — CHANGES_REQUESTED** [review]
Please have a look at the [best practices](https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices)

**Comment by dhilipkumars** [comment]
also bump the chart version

File: stable/ghost/templates/deployment.yaml
Comment (dhilipkumars) [inline stable/ghost/templates/deployment.yaml]: Even though this is not exactly as proposed, this still follows the same pattern as rest of the chart.   I think it's recommended to have ImageName , ImageTag and PullPolicy per container as its easy to configure in the future.

**Comment by dhilipkumars** [comment]
@kubernetes/charts-maintainers 
Quick Question:  A new container is added, should we ask the contributor to follow the below struct or allow to align it with existing chart variables?
```
volumePermision:
    image: 
         name: <IMAGE NAME>
         tag:  <IMAGE TAG>
         pullPolicy: <IMAGE PULL POLICY>
```

File: stable/ghost/templates/deployment.yaml
Comment (sameersbn) [reply stable/ghost/templates/deployment.yaml]: @dhilipkumars Bitnami's internal test system relies on the current naming of keys in `values.yml` file. Changing this will break our internal build/test pipeline. This is not something we can change with immediate effect.

Also I do not think there is any such policy for ordering keys in `values.yaml`

Comment (dhilipkumars) [reply stable/ghost/templates/deployment.yaml]: @sameersbn Thanks for the clarity, my concern is only about the new `initContainers` that's being introduced as a part of this PR not the existing container.  My other concern was if we do not get `initConainer` structured this time (when it is introduced) we may find it hard to bring it later as that mightbecome a breaking change then.  What do you think?

Comment (sameersbn) [reply stable/ghost/templates/deployment.yaml]: ok I misunderstood your request. it should be alright to have this structure for the newly added initContainer /cc @tompizmor

Comment (tompizmor) [reply stable/ghost/templates/deployment.yaml]: ok, I will update it!

**Review by dhilipkumars — APPROVED** [review]
Thanks for updating

**Comment by dhilipkumars** [comment]
LGTM from my side.
cc: @kubernetes/charts-maintainers could PTAL and approve if no issues.

**Comment by unguiculus** [comment]
Thanks @tompizmor and @dhilipkumars

**Pull request merged**
